ui/apps/platform/src/Containers/NetworkGraph/utils/networkGraphURLUtils.ts

```diff
@@ -21,3 +22,15 @@ export function getURLLinkToDeployment({
     const networkGraphLink = `${networkBasePath}/deployment/${deploymentId}${queryString}`;
     return networkGraphLink;
 }
+
+export function getPropertiesForAnalytics(searchFilter: SearchFilter) {
+    const cluster = searchFilter?.Cluster?.toString() || 'unknown cluster';
```

Comment:
why not just leave this one empty like the ns and deployments below?

Reply:
Good question. 
You can't run the simulator without choosing at least a cluster, but you can also run it with NS, and also deployments selected. So, empty strings for those last two are valid, but if the cluster is missing that's an error. So, I want to have a string that we can search on, and stands out in reports, in Amplitude for errors--"unknown cluster" was what I chose for that.